cyctl/cmd/get.go

```diff
@@ -10,14 +10,24 @@ import (
 
 var getCMD = &cobra.Command{
 	Use:   "get",
-	Short: "This is the get command",
-	Long:  "This is the get command",
+	Short: "Retrieve custom resources like modules, templates, and templateauthrules",
+	Long:  "getLong",
```

Comment:
Can you add a description here

Reply:
Yeah, i'm making all those changes, also added examples and i'm also including the refactoring #282

Reply:
Sure thing. You can do it in the same PR